netbox_onboarding/netbox_keeper.py

```diff
@@ -30,6 +31,24 @@
 PLUGIN_SETTINGS = settings.PLUGINS_CONFIG["netbox_onboarding"]
 
 
+def object_match(obj, search_array):
+    """Used to search models for multiple criteria."""
+    try:
+        result = obj.objects.get(**search_array[0])
+        return result
```

Comment:
Put some documentation how `search_array` is used - ie first element should be always exact match in strict strategy and so on